content/howto-simple_io.ipynb

```diff
@@ -0,0 +1,834 @@
+   "metadata": {},
+   "source": [
+    "We'll use the file `tmmx_2019.nc` containing the temperature data for the year 2019. You can download the file manually by going to "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "http://www.northwestknowledge.net/metdata/data/tmmx_2019.nc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "or use the complete url for the file in the `Dataset` function of the `netCDF4` module."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from netCDF4 import Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you have downloaded the file manually, remove the `#` from the line below and run it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#f = Dataset('tmmx_2019.nc')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to download it automatically, run the cell below (**You need to have an active internet connection for this to work**):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f = Dataset('http://thredds.northwestknowledge.net:8080/thredds/dodsC/MET/tmmx/tmmx_2019.nc'+'#fillmismatch', 'r', format=\"NETCDF4\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, the air temperature information (in Kelvin) can be accessed by"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "air_temperature = f.variables['air_temperature']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The data is stored as a three-dimensional array; in the first axis we have the day when the temperature was measured, and the second and third axes are for latitude and longitude, respectively."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "air_temperature.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that `air_temperature` is a [netCDF variable](https://www.unidata.ucar.edu/software/netcdf/docs/group__variables.html). We can read it as a NumPy array using the following syntax:\n",
+    "```python\n",
+    "temperatures = air_temperature[:]\n",
+    "```\n",
+    "\n",
+    "To simplify our example, we'll read only the temperature data between July 1st and July 31st, 2019. These are indexed by 181 and 211, respectively, in the first axis of our `air_temperature` array (remember that when slicing the stop value - 212 in the expression below - is not included in the final result.)"
```

Comment:
It might be good to show where 181 and 212 came from, or a way of finding them programmatically.